### PR TITLE
update(redis): switch from aioredis to redis library

### LINKED ISSUE #140: update(redis): Swith to redis-py instead of aioredis
aioredis is deprecated

redis storage plugin should use redis-py:
https://github.com/redis/redis-py#usage

ioredis is included since version 4.2: https://github.com/aio-libs/aioredis-py

### PR DESCRIPTION
**Pull request by pcanto-hopeit**
closes #140

### CODE REVIEW

File: plugins/build/ci-static-plugins.sh [hunk 1]
```diff
@@ -84,7 +84,7 @@ python3 -m flake8 --max-line-length=120 plugins/ops/log-streamer/src/hopeit/ plu
 code+=$?
 python3 -m pylint plugins/ops/log-streamer/src/hopeit/log_streamer/
 code+=$?
-
+echo $code
```
Comment (hopeit-git) [inline plugins/build/ci-static-plugins.sh:87]: I think this `echo` can be removed.

**Review by hopeit-git — APPROVED** [review]

**Comment by hopeit-git** [comment]
Version needs to be increased.

**Review by hopeit-git — CHANGES_REQUESTED** [review]
Version increase in version file is missing.

**Comment by leosmerling-hopeit** [comment]
TODOs:
1) Increase engine version
2) There are still references to aioredis in https://github.com/hopeit-git/hopeit.engine/blob/master/plugins/streams/redis/setup.py
3) Check also https://github.com/hopeit-git/hopeit.engine/blob/master/plugins/storage/redis/setup.py

**Review by hopeit-git — APPROVED** [review]

**Pull request merged**